shared/src/types.rs

```diff
@@ -526,6 +526,59 @@ pub struct ChangeString {
     new: Option<String>,
 }
```

Comment:
Is ChangeString still used?

Reply:
Ah thank you, I intended to remove this and forgot!